{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/forms.py

```diff
@@ -1,5 +1,7 @@
 from django.contrib.auth import forms as admin_forms
 from django.contrib.auth import get_user_model
+from allauth.account.forms import SignupForm
+from allauth.socialaccount.forms import SignupForm as SocialSignupForm
 from django.utils.translation import gettext_lazy as _
```

Comment:
isort fails because these imports aren't sorted:

```suggestion
from allauth.account.forms import SignupForm
from allauth.socialaccount.forms import SignupForm as SocialSignupForm
from django.contrib.auth import forms as admin_forms
from django.contrib.auth import get_user_model
from django.utils.translation import gettext_lazy as _
```